docs/sources/mimir/configure/configure-ingester-circuit-breakers.md

```diff
@@ -0,0 +1,83 @@
+---
+aliases:
+  - ../configuring/configuring-ingester-circuit-breakers/
+description: Learn how to configure ingester's circuit breakers.
+menuTitle: Ingester's circuit breakers
+title: Configure Grafana Mimir Ingester circuit breakers
+---
+
+# Configure Grafana Mimir Ingester circuit breakers
+
+## How do Grafana Mimir Ingester circuit breakers work?
+
+A request to a resource protected by a circuit breaker follows these steps:
+
+- The request tries to acquire a circuit breaker permit.
+- If the circuit breaker is open, no permit is acquired, and the request fails with an _circuit breaker open error_.
+- Otherwise, a circuit breaker permit is acquired, and the request is executed.
+  - If the request execution meets the circuit breaker failure condition, a failure is recorded within the circuit breaker.
+  - Otherwise, a success is recorded with the circuit breaker.
+
+Depending on configurable frequencies of successes and failures, the circuit breaker transits from one state to another.
+
+Grafana Mimir Ingester circuit breakers are an experimental feature.
+They independently protect Mimir's write and read paths from slow push and read requests.
+More precisely, Grafana Mimir Ingester distinguishes between _push requests circuit breakers_ and _read requests circuit breakers_.
+
+### Push requests circuit breakers
+
+Push requests circuit breakers follow the general procedure explained [above](#how-do-grafana-mimir-ingester-circuit-breakers-work), with the following remark: a push request meets the push requests circuit breaker failure condition if its duration is longer than the configurable maximum push request duration.
+
+### Read requests circuit breakers
+
+Read requests circuit breakers follow the general procedure explained [above](#how-do-grafana-mimir-ingester-circuit-breakers-work), with the following remarks:
+
+- in order to protect the write path as much as possible, ingesters do not allow read push requests if their push circuit breakers are open.
+  This means that before an ingester tries to acquire a read circuit breaker permit, it first checks if its push circuit breaker is open.
+  If it is the case, a circuit breaker open error is returned.
+  Otherwise, the ingester tries to acquire a read requests circuit breaker permit.
+- a read request that acquired a read requests circuit breaker permit meets the read requests circuit breaker failure condition if its duration is longer than the configurable read request maximum duration.
+
+## Grafana Mimir Ingester circuit breakers configuration
+
+Grafana Mimir Ingester circuit breakers can be configured by using the following configuration options.
```

Comment:
I don't think repeating the same sort of documentation that already exists for each config setting is useful. This also seems likely to go out-of-day. Maybe this could be a set of examples oriented around specific configurations you might want with a minimal set of flags required for that?